### PR TITLE
Added link in Go To Line

### LINKED ISSUE #5728: VS Command?
What's the command name of this feature?
I accidentally messed up the shortcut and don't know what's the command to set the shortcut to.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e5b0abc5-a74f-4138-7608-6b4215bca8c0
* Version Independent ID: a16608ea-41ce-f2c2-dfd3-8ed50a5a083c
* Content: [Go To Line - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/reference/go-to-line?view=vs-2019)
* Content Source: [docs/ide/reference/go-to-line.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/reference/go-to-line.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-general**
* GitHub Login: @TerryGLee
* Microsoft Alias: **tglee**

### PR DESCRIPTION
**Pull request by weitzhandler**
Fixes #5728.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@weitzhandler : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/ide/reference/go-to-line.md [hunk 1]
```diff
@@ -30,3 +30,4 @@ In the **Line number (1 - \<n>)** box, enter the number of the line in the activ
 - [Set bookmarks in code](../../ide/setting-bookmarks-in-code.md)
 - [Find and replace text](../../ide/finding-and-replacing-text.md)
 - [Features of the code editor](../../ide/writing-code-in-the-code-and-text-editor.md)
+- [Go To Command - Visual Studio](../../ide/reference/go-to-command)
```
Comment (TerryGLee) [inline docs/ide/reference/go-to-line.md:33]: @weitzhandler Thanks for this! Please if you would, to match our paradigm for links, remove the " - Visual Studio" from the link text? After that, it's good to go. :)

**Comment by PRMerger15** [comment]
@weitzhandler : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by TerryGLee** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
I revised the relative link because the target article is in the exact same folder as the updated article. It tests correctly in GitHub Preview; will check it on live as a follow-up.

**Comment by ktoliver** [comment]
Thanks @TerryGLee

**Pull request merged**